_topic_map.yml

```diff
@@ -183,6 +183,8 @@ Topics:
 - Name: Installing on GCP
   Dir: installing_gcp
   Topics:
+  - Name: Preparing to Install on GCP
```

Comment:
@GroceryBoyJr, I don't want to be _that guy_, but we're using sentence case for titles; Can you change to _Preparing to install on GCP_ instead?

Reply:
Done. And for the record...you be you, @jboxman !